camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java

```diff
@@ -534,12 +535,14 @@ private boolean checkExecutionErrors(final FileSystem fs,
       final List<Pair<EtlKey, ExceptionWritable>> errorsFromFile = fileEntry.getValue();
       if (errorsFromFile.size() > 0) {
         log.error("Errors from file [" + filePath + "]");
+        StatsdReporter.gauge(configuration, "camus-errors", 1L);
```

Comment:
I would not report this here. If we only report per ETL key that maps nicely to partitions, so we'll know how many partitions had trouble by simply doing the sum.

Reply:
makes sense, I'll keep etl only